src/goInstallTools.ts

```diff
@@ -25,7 +25,8 @@ function getTools(goVersion: SemVersion): { [key: string]: string } {
 		'go-outline': 'github.com/lukehoban/go-outline',
 		'go-symbols': 'github.com/newhook/go-symbols',
 		'guru': 'golang.org/x/tools/cmd/guru',
-		'gorename': 'golang.org/x/tools/cmd/gorename'
+		'gorename': 'golang.org/x/tools/cmd/gorename',
+		'godoc': 'golang.org/x/tools/cmd/godoc'
```

Comment:
This should go inside the if block `if (goConfig['docsTool'] === 'godoc')`, because when user choose `gogetdoc`, they need not install `godoc`